lte/gateway/c/core/oai/tasks/nas/esm/esm_ebr_context.c

```diff
@@ -108,8 +109,8 @@ ebi_t esm_ebr_context_create(
      * Check the total number of active EPS bearers
      */
     else if (esm_ctx->n_active_ebrs > BEARERS_PER_UE) {
-      OAILOG_WARNING(
-          LOG_NAS_ESM,
+      OAILOG_WARNING_UE(
+          LOG_NAS_ESM, ue_mm_context->emm_context._imsi64,
```

Comment:
why not using parameter emm_context directly ?

Reply:
Resolved